packages/suite/src/views/wallet/nfts/NftsTable/NftsRow.tsx

```diff
@@ -0,0 +1,296 @@
+                                                                    accountType:
+                                                                        account.accountType,
+                                                                },
+                                                            }),
+                                                        );
+                                                    }
+                                                },
+                                            },
+                                            {
+                                                label: <Translation id="TR_VIEW_IN_EXPLORER" />,
+                                                icon: 'arrowUpRight',
+                                                onClick: () => {
+                                                    window.open(
+                                                        getNftContractExplorerUrl(network, nft),
+                                                        '_blank',
+                                                    );
+                                                },
+                                            },
+                                        ],
+                                    },
+                                    {
+                                        key: 'contract-address',
+                                        label: <Translation id="TR_CONTRACT_ADDRESS" />,
+                                        options: [
+                                            {
+                                                label: (
+                                                    <Row gap={spacings.xxs}>
+                                                        {nft.contract}
+                                                        <Icon name="copy" size={14} />
+                                                    </Row>
+                                                ),
+                                                onClick: () => {
+                                                    dispatch(
+                                                        shouldShowCopyAddressModal
+                                                            ? showCopyAddressModal(
+                                                                  nft.contract || '',
+                                                                  'contract',
+                                                              )
+                                                            : copyAddressToClipboard(nft.contract),
+                                                    );
+                                                },
+                                            },
+                                        ],
+                                    },
+                                ].filter(category => category) as GroupedMenuItems[]
```

Comment:
This was needed in case of TokenRow (where I copied it from), I think there it was checking wether the array doesn't include falcy values, because there were some conditional items. Here I don't have any conditional items so it's not necessary, can be dropped